src/packages/client/src/__tests__/runtime-tests/filter-nullable/index.js

```diff
@@ -0,0 +1,21 @@
+const { PrismaClient } = require('@prisma/client')
+const assert = require('assert')
+
+module.exports = async () => {
+  const db = new PrismaClient()
+
+  const prisma = new PrismaClient()
+  const result = await prisma.sale.findMany({
+    where: {
+      resale: null
+    },
+  })
+
+  console.log(result)
```

Comment:
`result` should return 2 `Sale` items with ID `1` and `2`

Reply:
Note: We'll need to change this console.log to an assert here